projects/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -28,6 +28,7 @@
     <PackageOutputPath>..\..\packages</PackageOutputPath>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <CodeAnalysisRuleSet>RabbitMQ.ruleset</CodeAnalysisRuleSet>
+    <LangVersion>latest</LangVersion>
```

Comment:
I would not set the language version to latest. Because that allows you to use language features that might not be supported by the TFMs chosen here. If you leave it blank you get automatically the proper language version supported by the TFM intersection

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version

Reply:
Doesn't "The compiler accepts syntax from the latest released version of the compiler" mean it will always choose the latest possible whatever framework version we selected?

Reply:
No, it will choose the the latest version the compiler knows about, overriding the version that would have been picked by the compiler based on the TFM.

@danielmarbach is right that it should not be set to get the automatic behavior that we want here.

Reply:
Well that's wrong then, sorry. I'll update it in one of my next PRs. Thanks for the review